webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/eventlog/EventLogDetailsService.scala

```diff
@@ -138,7 +138,7 @@ trait EventLogDetailsService {
 
   def getChangeRequestDetails(xml:NodeSeq) : Box[ChangeRequestDiff]
 
-  def getWorkflotStepChange(xml:NodeSeq) : Box[WorkflowStepChange]
+  def getWorkflowStepChange(xml:NodeSeq) : Box[WorkflowStepChange]
```

Comment:
it's used in workflow plugin, if you change it, it's not binary compatible anymore. It needs to be done in 6.0 branch, before 6.0.3